### DISCUSSION TITLE
error} unsupported game version, or corrupted game detected

### DISCUSSION THREAD

**Discussion by garnishgod** [DaXcess/LCVR#250] [Q&A] [answered]
my game is up to date and not corrupted cause i am just running the vr mod by itself, currently on version v55 which i believe is up to date so idk. if someone has an answer that would be great
![image](https://github.com/user-attachments/assets/b2e0d348-2826-4a05-b834-274defe56f06)

**Comment by Killquota** [comment, marked as answer]
the mod itself is not yet compatible with the new verison of lethal company
